### PR TITLE
Ddcnls 5285

### PR DESCRIPTION
**Pull request by richardjamesbaconHMRC**
CY-2 taxcalc card

### CODE REVIEW

File: app/config/ConfigDecorator.scala [hunk 1]
```diff
@@ -39,6 +39,7 @@ class ConfigDecorator @Inject() (environment: Environment, configuration: Config
 
   val defaultOrigin = Origin("PERTAX")
 
+  //def currentLocalDate: LocalDate = new LocalDate(2019, 1, 1)
```
Comment (OktawianC) [inline app/config/ConfigDecorator.scala]: unused/commented code

File: conf/application.conf [hunk 2]
```diff
@@ -232,7 +232,7 @@ external-url {
     host = "http://localhost:9242"
   }
   taxcalc {
-    host = "http://localhost:9417"
+    host = "http://localhost:9242"
```
Comment (OktawianC) [inline conf/application.conf]: I think this should be restored to 9417

Comment (richardjamesbaconHMRC) [reply conf/application.conf]: No, 9242 is the pertax-depencencies pport.

Comment (OktawianC) [reply conf/application.conf]: OK

File: app/config/ConfigDecorator.scala (hunk 1 above)
Comment (richardjamesbaconHMRC) [reply app/config/ConfigDecorator.scala]: removed.

**Pull request closed without merging**